Feature/require

Zod- opened the pull request:
Closes #35

olbat commented:
Ok for Gloomclaw, it seems that I've made my run with outdated dumps (did not have access to internet) instead of generating fresh ones.

olbat commented:
I've made a fix, the script is now looking for both hash key and hash value before stating that a there is not translation.

What do you prefer: another pull request or a fixup commit ?

Immemor commented:
as u wish in fact. I have a little preference for a new branch to check the final diff easily.

olbat commented:
Here is the new pull request #38. I close this one.

Zod- committed:
Remove all globals from luacheck

Zod- committed:
Put event functions back into globals

Zod- committed:
Use new require style
This helps forcing the use of require since it's
technically needed. Luacheck will warn if require
hasn't been used like this and also warns about
requires that are not needed.
